src/core/opamVersionCompare.ml

```diff
@@ -15,28 +15,13 @@
 let is_digit = function
   | '0'..'9' -> true
   | _ -> false
-;;
 
 (* [skip_while_from i f w m] yields the index of the leftmost character
  * in the string [s], starting from [i], and ending at [m], that does
  * not satisfy the predicate [f], or [length w] if no such index exists.  *)
 let rec skip_while_from i f w m =
-  if i = m then i
+  if Int.equal i m then i
```

Comment:
Does the removal of polymorphic compare speeds up the comparison ?

Reply:
it does a little bit but i've done it mostly to clean things up

Reply:
In that case, it should be in its proper commit, if not part of the speedup

Reply:
it's already the case in this PR

Reply:
As you already did!